src/server/src/routes/facultyMember.ts

```diff
@@ -82,24 +84,34 @@ router.post(
  *          GET Request - Read - "GET /api/facultyMember/get-profile"
  ******************************************************************************/
 
-router.get('/get-profile', async (req: Request, res: Response) => {
-    const { id } = req.body;
-    try {
-        if (!id) {
-            return res.status(BAD_REQUEST).json({
-                error: errors.paramMissingError,
-            });
+router.get('/get-profile',
+    passport.authenticate('jwt', { session: false }),
+    async (req: Request, res: Response) => {
+        const { userId } = req.user as JWTUser;
+        if (!userId) {
+            return res
+                .status(UNAUTHORIZED)
+                .json({ error: 'You should log in to your account to view the page.' });
         }
 
-        const facultyMember = await getFacultyMemberProfile(id);
-        return res.status(OK).json({ facultyMember }).end();
-    } catch (error) {
-        logger.err(error);
-        return res
-            .status(INTERNAL_SERVER_ERROR)
-            .json(errors.internalServerError)
-            .end();
-    }
+
+        const { id } = req.body;
+        try {
+            if (!id) {
```

Comment:
if statements should be outside of try-catch block

Reply:
Done.